snap/snapcraft.yaml

```diff
@@ -181,9 +166,15 @@ parts:
       cp -vr ./script $CRAFT_PART_INSTALL/bin/
       chmod +x $CRAFT_PART_INSTALL/bin/script/otbr-firewall
 
+      OTBR_OPTIONS="-DCMAKE_INSTALL_LIBDIR=${SNAPCRAFT_PART_INSTALL}/lib/"
+      OTBR_OPTIONS+=" -DCMAKE_INSTALL_SYSCONFDIR=${SNAPCRAFT_PART_INSTALL}/etc/"
+      OTBR_OPTIONS+=" -DOT_CFLAGS=-DOPENTHREAD_POSIX_CONFIG_DAEMON_SOCKET_BASENAME=\"/run/snap.openthread-border-router/openthread-%s\""
+      # OTBR_OPTIONS+=" -DCMAKE_BUILD_TYPE=Debug"
+
```

Comment:
May be you can replace OT_CFLAGS with the following
-DOT_CFLAGS=-DOPENTHREAD_POSIX_CONFIG_DAEMON_SOCKET_BASENAME=\"/run/snap.$CRAFT_PROJECT_NAME/openthread-%s\"